madhav/src/main/java/com/lftechnology/phpjava/main/views/Menu.java

```diff
@@ -0,0 +1,21 @@
+package main.java.com.lftechnology.phpjava.main.views;
+
+import main.java.com.lftechnology.phpjava.main.Scanner;
+
+public class Menu {
+    public static String[] menu = { 
+            "1. Extract salutation from name", 
+            "2. Check sum of multiples", 
+            "3. Find sum of fibonaccis", 
+            "0. Exit" 
+            };
+
+    public static int getMenu() {
+        System.out.println("Please enter the action to perform:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)